google/cloud/bigtable/row_filters.py

```diff
@@ -494,7 +494,43 @@ class ExactValueFilter(ValueRegexFilter):
     def __init__(self, value: bytes | str | int):
         if isinstance(value, int):
             value = _PACK_I64(value)
-        super(ExactValueFilter, self).__init__(value)
+        elif isinstance(value, str):
+            value = value.encode("ascii")
```

Comment:
Note: the filter will raise an exception if the input string can not be  encoded to ascii. That was how I understood the docstring text. Let me know if that's how it should be